fyle_accounting_mappings/models.py

```diff
@@ -334,15 +334,16 @@ def bulk_create_or_update_destination_attributes(
                             id=primary_key_map[attribute['destination_id']]['id'],
                             value=attribute['value'],
                             detail=attribute['detail'] if 'detail' in attribute else None,
-                            active=attribute['active'] if 'active' in attribute else None
+                            active=attribute['active'] if 'active' in attribute else None,
+                            updated_at=datetime.datetime.now()
```

Comment:
can do
`from datetime import datetime`
and use datetime.now() instead of datetime.datetime.now()